From 7ab42400ac98ce8d0f720a00f8b977699ad6e9ca Mon Sep 17 00:00:00 2001
From: Christian Kohn <christian.kohn@xilinx.com>
Date: Wed, 11 Mar 2015 11:13:28 -0700
Subject: [PATCH 501/793] drm: xilinx: plane: Do not set background color in
 DRM_MODE_DPMS_ON

Don't reset the background color but use default value that the
user selects when instantiating the IP core. In the long run
this should be a separate plane property.

Signed-off-by: Christian Kohn <christian.kohn@xilinx.com>
Reviewed-by: Hyun Kwon <hyun.kwon@xilinx.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
---
 drivers/gpu/drm/xilinx/xilinx_drm_plane.c |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/drivers/gpu/drm/xilinx/xilinx_drm_plane.c b/drivers/gpu/drm/xilinx/xilinx_drm_plane.c
index 0b4876e..cb551f0 100644
--- a/drivers/gpu/drm/xilinx/xilinx_drm_plane.c
+++ b/drivers/gpu/drm/xilinx/xilinx_drm_plane.c
@@ -609,7 +609,6 @@ void xilinx_drm_plane_manager_dpms(struct xilinx_drm_plane_manager *manager,
 
 		if (manager->osd) {
 			xilinx_osd_disable_rue(manager->osd);
-			xilinx_osd_set_color(manager->osd, 0x0, 0x0, 0x0);
 			xilinx_osd_enable(manager->osd);
 			xilinx_osd_enable_rue(manager->osd);
 		}
-- 
1.7.5.4

